python/deltalake/table.py

```diff
@@ -295,6 +298,39 @@ def from_data_catalog(
             table_uri=table_uri, version=version, log_buffer_size=log_buffer_size
         )
 
+    @classmethod
+    def create(
+        cls,
+        table_uri: Union[str, Path],
+        schema: Union[pyarrow.Schema, DeltaSchema],
+        mode: Literal["error", "append", "overwrite", "ignore"] = "error",
+        partition_by: Optional[Union[List[str], str]] = None,
+        name: Optional[str] = None,
+        description: Optional[str] = None,
+        configuration: Optional[Mapping[str, Optional[str]]] = None,
+        storage_options: Optional[Dict[str, str]] = None,
+    ) -> "DeltaTable":
+        if isinstance(schema, DeltaSchema):
```

Comment:
Docstring here would be very useful - it would be auto-generated into the main docs too.